librealsense: add version 2.56.5

lia-viam opened the pull request:
### Summary
Changes to recipe:  Add **librealsense/~~2.55.1~~** and **2.56.5**

#### Motivation
Just doing a version bump for the versions of librealsense I use at work

#### Details
The following diff may be helpful in comparing the changes from the previous version (2.53.1) to 2.55.1:
https://github.com/IntelRealSense/librealsense/compare/v2.53.1...v2.55.1

##### Patches
- most of the patches are basically the same, one has been mainlined 
- there were a bunch of missing includes that only came up on ubuntu; those got their own patch which is the `0004` patch

##### Firmware
All of the firmware except for D4XX has been removed: https://github.com/IntelRealSense/librealsense/compare/v2.53.1...v2.55.1#diff-28458d177c92a2acd777c55bf847b3e2c9025fcbe2f269c2dfd28292d8ba78d7

##### rsutils
The utilities library has been renamed to `rsutils` and should now be linked against realsense, so it has been added in the package_info:
https://github.com/IntelRealSense/librealsense/compare/v2.53.1...v2.55.1#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aR60

---
- [x] Read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md)
- [x] Checked that this PR is not a duplicate: [list of PRs by recipe](https://github.com/conan-io/conan-center-index/discussions/24240)
- [x] Tested locally with at least one configuration using a recent version of Conan 
- - **built on arm64 Darwin and amd64 Linux**

### Edit

I have also added 2.56.5 as we are upgrading to that version at work. If it's a problem to add two versions in one PR maybe we can just do 2.56.5. There was an additional patch that has been mainlined and is no longer needed 


### Edit 2

per discussion below I have changed it to just 2.56.5 which is the latest stable release

perseoGI commented:
Hi @lia-viam,
Thank you for your contribution; we appreciate it!

Since some time now, we are moving in Conan Center Index towards a posture of keeping the recipes simple and maintaining only the latest versions of each recipe unless there is a very justified reason.
That is why I suggest adding only one version in a PR, preferably the latest one.
In the case of a recipe that requires many patches per version, like this one, it will keep the diff simple and make it easier for maintainers to review and merge your PR.

Also, check out the config.yml, which does not have the latest version in it!

Thank you again!

lia-viam commented:
@perseoGI thank you for the explanation and your review comment

I have updated the PR to only include version 2.56.5, which we use at work and which is also the latest stable (non-beta release). I have also updated the `config.yml`, thank you for catching that mistake!

I tested once again on arm64 darwin after making these changes

jcar87 commented on recipes/librealsense/all/conanfile.py:
```diff
@@ -136,6 +142,15 @@ def package_info(self):
+
+        # rsutils component
+        if Version(self.version) >= "2.56.5":
+            self.cpp_info.components["rsutils"].type = "static-library"
```
```suggestion
```
